db/migrate/20200429080939_report_entries_flags.rb

```diff
@@ -0,0 +1,5 @@
+class ReportEntriesFlags < ActiveRecord::Migration[6.0]
+  def change
+    add_column :preupgrade_report_entries, :flags, :string
```

Comment:
AFAIK `:string` is limited to 255 characters, is that enough? We use `:text` for `tags` and `detail`